DataFormats/METReco/src/CSCHaloData.cc

```diff
@@ -54,7 +54,7 @@ int CSCHaloData::NumberOfHaloTracks(HaloData::Endcap z) const
   int n = 0 ;
   for(unsigned int i = 0 ; i < TheTrackRefs.size() ; i++ )
     {
-      edm::Ref<reco::TrackCollection> iTrack( TheTrackRefs, i ) ;
+      edm::Ref<reco::TrackCollection> iTrack( TheTrackRefs[i] ) ;
```

Comment:
why was this a bug?
```  Ref(RefVector<C, T, F> const& refvector, key_type itemKey) ``` exists 
is the point that it's becoming a bug with updates of the interface?